packages/admin/admin/package.json

```diff
@@ -95,6 +95,7 @@
         "@emotion/react": "^11.5.0",
         "@emotion/styled": "^11.3.0",
         "@mui/material": "^6.0.0",
+        "@mui/utils": "^6.0.0",
```

Comment:
The changeset should mention this new peer dependency.

Reply:
I suppose we should add it in the upgrade script as well, right?

Reply:
Actually I don't think we need this as a peer dependency, as it's already a dependency of `@mui/material`. 
I removed it.

Reply:
I set `@mui/utils` as a normal dependency, as discussed